map.go

```diff
@@ -204,14 +204,13 @@ const (
 	defaultMaxBucketCapacity uint32 = 4096
 )
 
-func init() {
+func _() {
```

Comment:
I think we could get rid of the function now that we're not requiring code to run. That is, the variable declaration below could be defined at the file-level:

```
const expectedBucketSize = ptrSize + 6*4
var _ [0]struct{} = [unsafe.Sizeof(bucket[int, int]{}) - expectedSize]struct{}{}
```